### PR TITLE
Don't block on unavailable updates

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: src/power_bootloader.py [hunk 1]
```diff
@@ -258,30 +258,27 @@ def _bootload(_module, _module_address, filename, is_power_module):
                 bootload_8(_module_address, filename, power_communicator)
             elif not is_power_module and _module['version'] == POWER_API_12_PORTS:
                 bootload_12(_module_address, filename, power_communicator)
-            return True
+        except CommunicationTimedOutException:
+            logger.warning('E{0} - Module unavailable. Skipping...'.format(address))
         except Exception:
             logger.exception('E{0} - Unexpected exception during bootload'.format(address))
```
Comment (svanscho) [inline src/power_bootloader.py]: Add Skipping... in logs

File: src/power_bootloader.py [hunk 2]
```diff
@@ -258,30 +258,27 @@ def _bootload(_module, _module_address, filename, is_power_module):
                 bootload_8(_module_address, filename, power_communicator)
             elif not is_power_module and _module['version'] == POWER_API_12_PORTS:
                 bootload_12(_module_address, filename, power_communicator)
-            return True
```
Comment (svanscho) [inline src/power_bootloader.py]: Add log line that update went well

Comment (khenderick) [reply src/power_bootloader.py]: That's already logged in the `bootload_8` and `bootload_12` methods

Comment (svanscho) [reply src/power_bootloader.py]: Excellent

**Pull request merged**